### ISSUE TITLE
Uncontrolled re-run scripts

### ISSUE THREAD

**Issue by yangkerrr** [apommel/vscode-matlab-interactive-terminal#61]
When I finish running the matlab script in vscode, and then close vscode and reopen it, the terminal automatically runs the script. What is causing this problem? Is there a configuration I can change so that re-opening vscode will not re-run my program?

Thanks.

**Comment by WalkerSyz** [comment]
I also have this problem. Have you solved it?

**Comment by ZIYU18** [comment]
> I also have this problem. Have you solved it?

yes,I also have this problem,so i would like to ask how to sovle it

**Comment by ZIYU18** [comment]
> I also have this problem. Have you solved it?

I have found some methods to solve it,you can open settings to search  for " terminal.integrated.persistentSession" and turn off the " Enable Persistent Sessions" and "Persistent Session Revive Process" ,hope to help you !

**Comment by WalkerSyz** [comment]
> > I also have this problem. Have you solved it?
> 
> I have found some methods to solve it,you can open settings to search for " terminal.integrated.persistentSession" and turn off the " Enable Persistent Sessions" and "Persistent Session Revive Process" ,hope to help you !

Thank you for your suggestion. Recently, I have found that using the official MATLAB plugin in VSCode can effectively solve this problem. But I think it's better to use MATLAB itself.
